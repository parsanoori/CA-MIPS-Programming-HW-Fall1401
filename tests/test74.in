10
724901542
83011095
497993025
677022433
482467687
816595679
382856943
831236799
814064110
321991141