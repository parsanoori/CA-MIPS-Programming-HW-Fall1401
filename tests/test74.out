83011095
321991141
382856943
482467687
497993025
677022433
724901542
814064110
816595679
831236799